File: week04/day01/src/Sharpie.java
/**
 * Created by Bálint on 2017. 04. 03..
 */
public class Sharpie {
  String color;
  float width;
  float inkAmount;

  public Sharpie(String color, float width) {
    this.color = color;
    this.width = width;
    inkAmount = 100;
  }

  public void use() {
    inkAmount --;
  }

  public static void main(String[] args) {
    Sharpie sharpie = new Sharpie("Brown", 12);
    sharpie.use();
    System.out.println(sharpie.inkAmount);
  }
}

File: javascript_practice/10-advancedjs/sharpie.js
function Sharpie(color, width) {
  this.color =  color;
  this.width = width;
  this.inkAmount = 100;
  this.use =  function() {
    this.inkAmount -= width;
  };
}

let redSharpie = new Sharpie('red', 10);
console.log(redSharpie.inkAmount);

redSharpie.use();
console.log(redSharpie.inkAmount);
